setup-and-run.sh

```diff
@@ -16,6 +16,11 @@ if [[ $STRICT_SCRIPT =~ $TRUE_REG ]]; then
     set -o pipefail
 fi
 
+# export telemetry variables
+source /build.info
+export LT_PACKAGE="box"
```

Comment:
Just in case:
```
export LT_PACKAGE=${LT_PACKAGE:-box}
```
Currently we don't have anything depending to Box, but you never know.